sdk/sdktypes/id.go

```diff
@@ -27,6 +28,8 @@ type ID interface {
 	// Value returns the id value, meaning without the prefix.
 	Value() string
 
+	UUIDValue() *UUID
```

Comment:
why not just return `UUID`? why do we need the ptr?

Reply:
it makes life easier when the UUID is optional, like in db, I can just use the UUIDValue() instead of checking if it is zero value and then assigning null in the db

Reply:
in this case return an empty array

Reply:
fixed

PTAL